src/tools/dart-fix.md

```diff
@@ -31,4 +28,62 @@ To apply the proposed changes, use the `--apply` flag:
 $ dart fix --apply
 ```
 
+## Customizing dart fix
+
+The `dart fix` command applies its fixes
+based on the analysis configuration of the code it runs on. 
+To learn more about customizing the analysis,
+see [Customizing static analysis](/guides/language/analysis-options).
+
+### Example
+
+Imagine you have code like this:
+
+```dart
+class Vector2d {
+  final double x, y;
+  Vector2d(this.x, this.y);
```

Comment:
Consider adding a blank line between members to be consistent with the rest of the example code in this section.